features/support/overwrite_constants.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# Idea from
+# https://makandracards.com/makandra/6125-how-to-overwrite-and-reset-constants-within-cucumber-features
+#
+module ConstantsHelper
+  def saved_constants
+    @saved_constants ||= {}
+  end
+
+  def undefined_constants
+    @undefined_constants ||= {}
+  end
+
+  def overwrite_constant(constant, value, object = Object)
```

Comment:
Is it possible to use [`stub_const` from RSpec?](https://relishapp.com/rspec/rspec-mocks/docs/mutating-constants)

`cucumber/rspec/doubles` is already required in `features/support/env.rb` and I think that would make it available in Cucumber tests (although I haven't tried it).

Reply:
I've had a look at this and it appears that `stub_const` doesn't work here.